mopidy/core/playlists.py

```diff
@@ -172,15 +169,21 @@ def filter(self, criteria=None, **kwargs):
 
         .. deprecated:: 1.0
             Use :meth:`as_list` and filter yourself.
+
+        .. deprecated:: 1.1
+            Providing the criteria via ``kwargs`` is no longer supported.
```

Comment:
Oh yes :-) I can probably just remove this again since I've fixed the tests I wanted to fix by now.